pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -30,30 +30,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"kubevirt.io/client-go/api"
-
 	v1 "kubevirt.io/api/core/v1"
 	kubevirtfake "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/fake"
 
+	"kubevirt.io/kubevirt/pkg/libvmi"
```

Comment:
Please consider squashing this commit with the one introducing the fake client.

Reply:
Done.